### PR TITLE
Update pretraining task

### PR DESCRIPTION
**Pull request by boru-roylu**
1. Add Gradient Accumulation to use 4096 batch size.
2. Add the function for ViT initialization for pretraining.
3. Rename `tasks/mtm.py` to `tasks/pretraining.py`.
4. Fix some nits and update docstring.

### CODE REVIEW

**Review by santiaontanon-google — APPROVED** [review]
Thanks Roy, LGTM!

**Review by jtainslie — APPROVED** [review]
Readability lgtm.

**Review by wqfcr-google — APPROVED** [review]
